uploadNeuroDB/file_insertion.pl

```diff
@@ -0,0 +1,509 @@
+
+# Generic TODOs:
+
+##TODO 1: once ExitCodes.pm class merged, replace exit codes by the variables
+# from that class
+
+
+use strict;
+use warnings;
+use Getopt::Tabular;
+use File::Basename;
+use File::Temp qw/ tempdir /;
+use Date::Parse;
+
+###### Import NeuroDB libraries to be used
+use NeuroDB::DBI;
+use NeuroDB::MRI;
+use NeuroDB::File;
+use NeuroDB::MRIProcessingUtility;
+##TODO 1: add line use NeuroDB::ExitCodes;
+
+
+##TODO 1: move those exit codes to ExitCodes.pm
+my $INVALID_SCANNER_ID     = 140; # new
+my $PROTOCOL_ID_NOT_FOUND  = 141; # new
+my $GET_SESSION_ID_FAILURE = 142; # new
+my $INVALID_UPLOAD_ID      = 143; # new
+my $GET_SUBJECT_ID_FAILURE = 96;  # same as MRIProcessingUtility.pm
+my $CANDIDATE_MISMATCH     = 111; # same as minc_insertion.pl
+my $FILE_NOT_UNIQUE        = 6;
+
+###### Table-driven argument parsing
+
+# Initialize variables for Getopt::Tabular
+my $profile       = undef;
```

Comment:
Variables declared with `my` are initialized to `undef` automatically: you do not have to do this yourself.

Reply:
All changed. Thanks!